.github/workflows/build_commits.yaml

```
@@ -0,0 +1,42 @@
+name: build_commits
+
+
+on: [pull_request]
+
+jobs:
+  build_commits:
+    name: build commit code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2.1.3
```

To have better control over the workflows, we recommend pinning the actions to SHA rather than tags. Could you update the actions referencing tags to their corresponding SHA?
For example, here it would be:
```suggestion
        uses: actions/setup-go@37335c7bb261b353407cff977110895fa0b4f7d8
```